ERCS/erc-8189.md

```diff
@@ -0,0 +1,115 @@
+---
+eip: 8189
```

Comment:
```suggestion
eip: 8114
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.  You don't get to pick your own.

Please also update the filename.